threshsign/src/bls/relic/BlsThresholdSigner.cpp

```diff
@@ -98,6 +98,7 @@ void BlsThresholdSigner::deserializeDataMembers(istream &inStream) {
   deserialize(inStream, id_);
   BNT idNum(id_);
   idNum.toBytes(serializedId_, sizeof(id_));
+  delete params;
```

Comment:
This fix works, but it doesn't make the code exception safe. We probably want to look at using unique_ptrs everywhere instead of this in future commits. 

Same is true everywhere else we use delete outside a destructor also. We'll need to clean up the memory management. Fine to leave as is for now.

Reply:
I second this. Smart pointers is the only way forward.

Reply:
Definitely Smart pointers are the right way to go. Here I try to fix the current leaks without major changes, might be a separate task should be made to do the conversion from raw pointers to smart pointers?

Reply:
Totally. No need to make that change in this commit.